man/systemd.netdev.xml

```diff
@@ -450,6 +450,13 @@
           This setting is compulsory.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Protocol=</varname></term>
+        <listitem>
+          <para>Allows setting the protocol used for VLAN. Takes 802.1q or,
+          802.1ad, and defaults to unset and kernel's default is used.</para>
```

Comment:
Please use `<literal>` tag for `802.1q` and `802.1ad`.

Maybe, `for VLAN.` -> `for the VLAN interface.`